deployments/types-rococo.json

```diff
@@ -1,16 +1,17 @@
 {
   "--1": "Rococo Types",
+  "RococoAddress": "AccountId",
```

Comment:
As i removed the `Address` type from `common.js`, my understanding is that Rococo and Wococo contain the same `Address` type as Millau . In case this is wrong please let me know and i will revert the change or adjust it accordingly.